docs/format/db.rst

```diff
@@ -4,18 +4,31 @@
 Introduction
 ------------
 
-This section documents how prjxray represents the bitstream database
+This section documents how prjxray represents the bitstream database. The databases are plain text files, using either simple line-based syntax or JSON. The databases are located in `database/<device_class>/`. The `settings.sh` file contains some configurations used by the tools that generate the database, including the region of interest (ROI, see [[Glossary]]).
 
 These ".db" files come in two common flavors:
  * `segbits_*.db`_: encodes bitstream bits
  * `mask_*.db`_: which bits are used by a segment? Probably needs to be converted to tile
 
 Also note: .rdb (raw db) is a convention for a non-expanded .db file (see below)
 
+Segment bit positions
+---------------------
+
+Bit positions within a segment are written using the following notation: A two digit decimal number followed by an underscore followed by a two digit decimal number. For example `26_47`.
+
+The first number indicates the frame address relative to the base frame address for the segment and ranges from `00` to `35` for Atrix-7 CLB segments.
+
+The second number indicates the bit position with
```

Comment:
Incomplete sentence? We've had a couple questions around this, probably good to get a complete description of this that we can point people too.  This is probably a good place to talk about tile grid and how it applies to segbit.

Reply:
I added a FIXME.